events/wmt20.md

```diff
@@ -217,3 +217,46 @@ Friday, November 20, 2020
 | | [***The Tatoeba Translation Challenge – Realistic Data Sets for Low Resource and Multilingual MT***](https://aclanthology.org/2020.wmt-1.139.pdf) <br>Jörg Tiedemann |
 | | [***Human-Paraphrased References Improve Neural Machine Translation***](https://aclanthology.org/2020.wmt-1.140.pdf) <br>Markus Freitag, George Foster, David Grangier and Colin Cherry |
 | |	[***Incorporating Terminology Constraints in Automatic Post-Editing***](https://aclanthology.org/2020.wmt-1.141.pdf) <br>David Wan, Chris Kedzie, Faisal Ladhak, Marine Carpuat and Kathleen McKeown |
+
+## Results
+
+#### Full results of the shared tasks: [***Findings of the 2020 Conference on Machine Translation (WMT20)***](https://aclanthology.org/2020.wmt-1.1.pdf)
+
+### News Translation
+
+#### → English
+
+| Language pair | Annotation style | Average score | Average z-score (standard score) | System |
+| --- | --- | --- | --- | --- |
+| Chinese→English | Segment rating with document context | 77.5 | 0.102 | `VolcTrans` |
```

Comment:
Generally, if a column has the same value for all rows, it should not be in the table.

Reply:
There are two types of annotation styles used here  - segment rating with and without document context, so I've put it in a separate column and included it in the table. Otherwise, I can suggest separating the results for each table with short descriptions about the change of annotation style.